packages/react-components/src/components/VideoGallery/styles/FloatingLocalVideo.styles.ts

```diff
@@ -38,6 +38,12 @@ export const SMALL_FLOATING_MODAL_SIZE_PX = { width: 58, height: 104 };
  */
 export const LARGE_FLOATING_MODAL_SIZE_PX = { width: 215, height: 120 };
 
+/**
+ * Vertical gallery floating modal width and height in rem
+ * Aspect ratio: 16:9
+ */
+export const VERTICAL_GALLERY_FLOATING_MODAL_SIZE_PX = { width: 144, height: 81 };
```

Comment:
should this be 144 x 90? I guess thats not 16: 9 hey?

Reply:
Oh I just followed the size in Figma and it is the following currently:
![image](https://user-images.githubusercontent.com/79475487/221067303-86f3fe18-cb8b-45a1-bd89-a28e9fd33a4e.png)